.env

```diff
@@ -1,2 +1,2 @@
-secretKey = AwNddQ13R,
+SECRETKEY=AwNddQ13R,
```

Comment:
Files containing sensitive information should NOT be checked into version control (an exception maybe private repositories). I believe this belongs in your `.gitignore` file.